repobee_sanitizer/_sanitize_repo.py

```diff
@@ -128,9 +162,28 @@ def _sanitize_files(
             LOGGER.info(f"Shredded file {relpath}")
 
 
+def _format_error_string(files_with_errors: [_FilesWithErrors]) -> str:
```

Comment:
I'm guessing this is a list?

```suggestion
def _format_error_string(files_with_errors: List[_FileWithErrors]) -> str:
```